### ISSUE TITLE
Migrate the documentation to readthedocs.org

### ISSUE THREAD

**Issue by Tzaphkiel** [sdmx-twg/sdmx-csv#7]
Decision has been taken to migrate the documentation to readthedocs.org.

We should take this opportunity to consolidate and improve the current documentation.
This repository will details the CSV format part of the full documentation glued together in sdmx-im.

**Comment by pedroacarranza** [comment]
@dosse @agent96 Is this covered by the work being done by HMS?

**Comment by dosse** [comment]
Yes, this was supposed to achieve the same objective: easy management of the technical standards inside GitHub using md pages and a tool to publish them as a nicely readable web site.

**Comment by Tzaphkiel** [comment]
So, should this issue not be closed?

**Comment by dosse** [comment]
> So, should this issue not be closed?

No, let's continue to use it for the currently ongoing migration using MkDocs. I changed the title accordingly. --> @jr-hms
